src/jobs/index.md

```diff
@@ -11,3 +11,4 @@ We currently have the following job openings on the Flutter and Dart teams:
 * [Senior Technical Program Manager](/jobs/tpm)
 * [Developer Relations Engineer](/jobs/dre)
 * [Dart Native Runtime Engineer](/jobs/native_runtime)
+* [Flutter Engineering Productivity](/jobs/infrastructure.html)
```

Comment:
`s/.html//`

The working target URL is [/jobs/infrastructure](https://flutter.dev/jobs/infrastructure)

Reply:
Thanks, removed the .html suffix.

Reply:
@domesticmouse I believe we'll need your approval to remove the requested changes flag. Thanks!